app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -195,7 +195,11 @@ public void onContextMenu(GeckoSession session, int screenX, int screenY, String
                     if (elementSrc != null && uri != null) {
                         callback.onLongPress(new HitTarget(true, uri, true, elementSrc));
                     } else if (elementSrc != null) {
-                        callback.onLongPress(new HitTarget(false, null, true, elementSrc));
+                        if (elementSrc.endsWith("jpg") || elementSrc.endsWith("gif") ||
+                                elementSrc.endsWith("tif") || elementSrc.endsWith("bmp") ||
+                                elementSrc.endsWith("png")) {
+                            callback.onLongPress(new HitTarget(false, null, true, elementSrc));
+                        }
```

Comment:
I asked snorp whether GeckoView can tell us what the element is. They will add an API for us - I filed a bug for that: https://bugzilla.mozilla.org/show_bug.cgi?id=1445240

I'll merge this here as a workaround and file a new issue to clean that up once the new API is available.

Reply:
Filed #2283